src/testUtils/mocks/dataSources.ts

```diff
@@ -1445,7 +1445,7 @@ const createMockEnum = (enumValue?: string | Record<string, Codec | Codec[]>): E
     codec[`as${upperFirst(key)}`] = enumValue[key];
   }
 
-  return createMockCodec(codec, false) as Enum;
+  return createMockCodec(codec, !enumValue) as Enum;
```

Comment:
Why this change? I think it makes sense, but what triggered it?

Reply:
This was required to test isEmpty condition for the signatory. 

It was always giving `false` and I wanted it to be `true` while mocking the targetSignatory returned by `authorizationsGiven`, so that it can be verify that the `getOne` from `authorizations` is being called i